content/en/docs/components/training/tftraining.md

```diff
@@ -432,12 +432,12 @@ A TFJob has a TFJobStatus, which has an array of TFJobConditions through which t
 
 Success or failure of a job is determined as follows
 
-* If a job has a **chief** success or failure is determined by the status
+* If a job has a **chief**, success or failure is determined by the status
   of the chief.
-* If a job has no chief success or failure is determined by the workers.
-* In both cases the TFJob succeeds if the process being monitored exits
+* If a job has no chief, success or failure is determined by the workers.
+* In both the cases, the TFJob succeeds if the process being monitored exits
```

Comment:
```suggestion
* In both cases the TFJob succeeds if the process being monitored exits
```

"In both the cases" may be grammatically incorrect. I recommend reverting this particular change.

Thanks for the other comma suggestions.

cc @RFMVasconcelos @Bobgy

Reply:
done.